src/command/use_.rs

```diff
@@ -85,6 +86,17 @@ Options:
             Use::Yarn(spec) => session.pin_yarn_version(&spec)?,
             Use::Other { name, .. } => throw!(NoCustomUseError::new(name)),
         };
+        if let Some(project) = session.project() {
+            let errors = project.autoshim();
+
+            for error in errors {
+                if error.is_user_friendly() {
+                    display_error(ErrorContext::Notion, &error);
+                } else {
+                    display_unknown_error(ErrorContext::Notion, &error);
+                }
+            }
+        }
```

Comment:
Again here, I think these should be displayed to the user as warnings instead of errors, since autoshimming was not  explicitly invoked by the user in this context, and they don't affect the actual command status.

Reply:
Auto-shimming will never be invoked by the user, because it is an implicit action.  I was actually on the fence as to whether or not to have `notion use` fail when errors occur while auto-shimming, but decided to keep it consistent with the behavior of tool shims.

Reply:
This was resolved in last week's call.